docs/components/integration-framework/connectors/out-of-the-box-connectors/aws-lambda.md

```diff
@@ -0,0 +1,49 @@
+---
+id: awslambda
+title: AWS Lambda Connector
+description: Invoke AWS Lambda function.
+---
+
+The AWS Lambda Connector allows you to connect your BPMN service with the Amazon Web Service's [AWS Lambda Service](https://aws.amazon.com/lambda/).
+
+## Prerequisites
+
+To use an **AWS Lambda Connector**, you need to have an [AWS Lambda Function](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html), IAM key, and secret pair with permissions for execute function. See the [AWS Lambda developer guide](https://docs.aws.amazon.com/lambda/latest/dg/lambda-permissions.html) to learn more.
```

Comment:
"...and secret pair with permissions for execute function" -- do you mean "...and secret pair with permissions for the execute function" or "to execute the function?"